### ISSUE TITLE
Make training loss configurable via config and default to MSE

### ISSUE THREAD

**Issue by marianovitasari20** [alan-turing-institute/ice-station-zebra#275]
### Problem / Goal

Currently, the default training loss is MAE and it is not properly configurable through the config file. Changing the loss function requires modifying `base_model` directly, which reduces flexibility and makes experimentation more cumbersome.

In addition, MAE may not be the most suitable default for many deep learning optimisation workflows. MSE is generally a better default choice due to its smoother differentiability properties and more stable optimisation behaviour.

### What does good look like?

- [ ] Making the training loss configurable via the config file
- [ ] Changing the default loss from MAE to MSE

**Comment by jemrobinson** [comment]
Duplicate of #53
